interfaces/hotplug/deviceinfo.go

```diff
@@ -89,3 +90,43 @@ func (h *HotplugDeviceInfo) Attribute(name string) (string, bool) {
 	val, ok := h.Data[name]
 	return val, ok
 }
+
+func (h *HotplugDeviceInfo) firstAttrValueOf(tryAttrs ...string) string {
+	for _, attr := range tryAttrs {
+		if val, ok := h.Attribute(attr); ok && val != "" {
```

Comment:
Why are we testing both ok and value being non-empty?

Reply:
Fair point, it's a bit excessive. Changed to just check if value is not empty.